packages/babel-plugin/src/utils/ast.tsx

```diff
@@ -128,6 +130,29 @@ export const getInterpolation = <TNode extends {}>(
     const { accessPath, bindingIdentifier } = getMemberExpressionMeta(expression);
     const binding = meta.parentPath.scope.getBinding(bindingIdentifier.name);
 
+    if (
+      binding &&
+      binding.path.isImportDefaultSpecifier() &&
+      binding.path.parentPath.isImportDeclaration()
+    ) {
+      const moduleImportName = binding.path.parentPath.node.source.value;
+      const modulePath = require.resolve(meta.state.cwd + '/' + moduleImportName);
+      const moduleCode = fs.readFileSync(modulePath, 'utf-8');
+      const ast = parse(moduleCode, { sourceType: 'module' });
```

Comment:
Should we cache this `ast` here because we might end up reading same `modulePath` many times for eg.

**module C** is required by both **module A & B**. In this case we might end up parsing **module C** twice. 🤔

Reply:
oh definitely - just not bothered in the spike

Reply:
let me write up a description of what we want to solve in the spike